### DISCUSSION TITLE
SEVI中level3的A的问题

### DISCUSSION THREAD

**Discussion by meishuyao** [physics-data/faq#123] [Q&A] [answered]
一个电子经过两次倍增，产生的电子数在10^6数量级，但是A还是255饱和对吗，或者说亮斑中心一片全都是饱和的。

另外level3中是否不用考虑level2中的高斯分布积分的问题了，直接计数就行。

@littlewu2508 谢谢助教

**Comment by meishuyao** [comment]
另外请问

从第一层 MCP 出射后，大量电子会在两层 MCP 之间的间隙飞行，间隙距离为 **$d_1=10^{-5}$**，飞行后电子入射第二层 MCP，同样有 10% 的电子损失，以及 N ~ $\pi(1000)$ 的放大倍数，以及相同的出射角度分布。最终，从第二层 MCP 出射后，距离荧光屏还有 **$d_2=10^{-3}$** 的飞行距离，最终 CCD 每个像素数值，是落入该像素内的电子数目。

其中d1和d2的单位与R=0.8的单位是一样的吗

**Comment by meishuyao** [comment, marked as answer]
考虑到12中的A=100，是否level3中每一层MCP，N ~ $ \pi (10) $ 会更合理，而不是 N ~ $\pi(1000)$ 。

**Reply by meishuyao** [reply]
尽管网上的资料显示，2层MCP的增益确实在10^6数量级，但我猜测CCD不能探测到单个的电子。所以我认为尽管与实际不符，但在这里让A=100会更合理

**Reply by littlewu2508** [reply]
CCD 是像素化的，一个亮斑包含多个像素点，虽然一个亮斑有百万个电子，每个像素点落入的电子数目没有 10^6 个。不过我自己做了一下，发现确实每个像素内落入的电子数大概有 5000 个，所以应该再除以 50 才行（相当于荧光屏+CCD能够探测到 50 个电子）。
